web/src/views/store.coffee

```diff
@@ -82,6 +129,7 @@ export git_run_log = (###* @type string ### log_args) =>
 	]
 	return if not log_data?
 	parsed = parse log_data, branch_data, stash_data, sep
+	apply_mailmap parsed.commits
```

Comment:
isn't an `await` missing here? Intuitively I'd say this doesn't even work without it